tests/offline.spec.ts

```diff
@@ -26,8 +37,20 @@ test("Works offline", async ({ page }) => {
   await goOffline();
 
   await page.getByTestId("record").click();
-  await expect(page.locator("h1")).toContainText("Record vaccinations");
+  await expect(page.getByTestId("child-status").nth(0)).toContainText(
+    "Not yet"
+  );
 
   await page.getByTestId("child-link").nth(0).click();
   await expect(page.getByTestId("full-name")).toContainText("Aaron Pfeffer");
+
+  await page.getByTestId("confirm-button").click();
+
+  await goOnline();
```

Comment:
At this point I'm not really familiar with this part of the system and I'm left wondering whether the "Vaccinated" field should be `Vaccinated` or `Not Yet`. Especially as the page gets reloaded further down. Would it make sense to add a test here to make that clear?

Reply:
And by test, I mean an expectation. e.g.:

```
  await expect(page.getByTestId("child-status").nth(0)).toContainText(
    "Vaccinated"
  );
```

Reply:
It will be `Vaccinated`, shimmed by the client-side JS, but we don't implement that just yet; or the current implementation needs some updating. Can we add this as a follow-up? I'm expecting to wrap things up on the spiking and then card out stories to implement more of the prototype.